docs/source/app-dev/bindings-x-lang/index.rst

```diff
@@ -22,12 +22,10 @@ Note that for different reasons we chose codegen, but that is not the only optio
 
 - `Template Haskell <https://wiki.haskell.org/Template_Haskell>`_
 
-- Scala macro annotations (not future-proof enough to use when implementing the last Scala codegen)
-
 Building Ledger Commands
 ========================
 
-No matter what approach you take, either manually building commands or writing a codegen to do this, you need to understand how ledger commands are structured. This section demonstrates how to build create and exercise commands manually and how it can be done using contract classes generated by Scala codegen.
+No matter what approach you take, either manually building commands or writing a codegen to do this, you need to understand how ledger commands are structured. This section demonstrates how to build create and exercise commands manually and how it can be done using contract classe.
```

Comment:
```suggestion
No matter what approach you take, either manually building commands or writing a codegen to do this, you need to understand how ledger commands are structured. This section demonstrates how to build create and exercise commands manually and how it can be done using contract classes.
```